ios/Classes/PosthogFlutterPlugin.swift

```diff
@@ -19,6 +19,9 @@ public class PosthogFlutterPlugin: NSObject, FlutterPlugin {
         registrar.addMethodCallDelegate(instance, channel: channel)
     }
 
+    private let dispatchQueue = DispatchQueue(label: "com.posthog.PosthogFlutterPlugin",
+                                              target: .global(qos: .utility))
+
     public static func initPlugin() {
         let autoInit = Bundle.main.object(forInfoDictionaryKey: "com.posthog.posthog.AUTO_INIT") as? Bool ?? true
```

Comment:
Not sure I understand the question, can you clarify?

Reply:
not sure because this is past me but I believe this is to have the SDK enabled before the Flutter engine is up and running so can do more as an SDK.
I also don't like this approach (duplication) but it's possible to opt-out via the manual setup https://posthog.com/docs/libraries/flutter